sass/true/_context.scss

```diff
@@ -87,8 +87,8 @@ $_true-output-context: ();
 /// Validate the new context, and return an updated context value
 /// @access private
 /// @group private-context
-/// @param {'assert' | 'output' | 'expect' | null} $new -
-///   Add a new `assert`, `output`, or `expect` layer
+/// @param {'assert' | 'output' | 'expect' | 'contains' | null} $new -
+///   Add a new `assert`, `output`, `expect`, or `contains` layer
```

Comment:
I may be mistaken, but I think we also want this type of change on L79, L106, L120, L121, L129, and L132.

Reply:
Again, correct me if I'm wrong, but I think this change (adding `contains`) should also happen on L64, L79, L100, L120, and L121.